### DISCUSSION TITLE
Emporia Vue2 - Home Assistant ESPHome  -   CMake Error

### DISCUSSION THREAD

**Discussion by CavemanTitan** [emporia-vue-local/esphome#241] [Q&A]
Getting a new error when i try to install the YAML from the code page into my home assistant esphome project. 
ESPHome version 11.6, 
Notably I see the following issue, but ill attach the log as well. 

```
-- Check for working C compiler: /data/cache/platformio/packages/toolchain-xtensa-esp32/bin/xtensa-esp32-elf-gcc -- broken
-- Configuring incomplete, errors occurred!
See also "/data/build/emporiavue2/.pioenvs/emporiavue2/CMakeFiles/CMakeOutput.log".
See also "/data/build/emporiavue2/.pioenvs/emporiavue2/CMakeFiles/CMakeError.log".

fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
CMake Error at /data/cache/platformio/packages/tool-cmake/share/cmake-3.16/Modules/CMakeTestCCompiler.cmake:60 (message):
  The C compiler
    "/data/cache/platformio/packages/toolchain-xtensa-esp32/bin/xtensa-esp32-elf-gcc"
is not able to compile a simple test program.
It fails with the following output:
```

 Change Dir: [logs_emporia-vue_run (1).txt](https://github.com/emporia-vue-local/esphome/files/13503614/logs_emporia-vue_run.1.txt)

**Comment by flaviut** [comment]
Looks like this is the same issue as https://github.com/emporia-vue-local/esphome/discussions/105 -- any chance you're able to test on a PC?

Closing to keep this discussion in one place.

**Reply by CavemanTitan** [reply]
I was using HomeAssistant on a desktop, but if you asking about the python tools installed onto a desktop, no I struggled getting that working.

**Reply by flaviut** [reply]
The logs look like they were made on an ARM machine.

I unfortunately can't help with getting set up with dev tools, but the ESPHome discord or forums are likely a good place to ask those kinds of questions since they are not vue-specific.
